MSProject/ViewControllers/DetailBook/WBDetailBookViewController.swift

```diff
@@ -126,17 +125,15 @@ class WBDetailBookViewController: UIViewController {
         MBProgressHUD.showAdded(to: _view, animated: true)
         viewModel.loadComments(book: bookViewModel.book).startWithResult { [unowned self] result in
```

Comment:
if you are only interested in the errors you can use startWithFailed

Reply:
not only the error, I just dont care the response of success, but when completion (success or failure) I will call another service... or we can synch those services in a new way? =) I'm eager to learn ^^

Reply:
so, even if the comments fail to load you want to load the suggestions? if that is the case then why do you need to wait for the comments request to end to make the other one? can't you make both at the same time?